app/views/application/review.html

```diff
@@ -116,7 +116,45 @@ <h3 class="govuk-heading-m">References to be requested if you accept an offer</h
     {% include "_includes/review/safeguarding.html" %}
   </section>
 
-  {{ govukButton({
-    text: "Continue"
-  }) if not closed }}
+  {% set dateToday = "_" | dateNow | date %}
+  {% set canSubmit = true %}
+  {% set cannotSubmitReasons = [] %}
+
+  {% set choices = applicationValue("choices") | toArray %}
+  {% for choice in choices %}
+    {% set dateCourseOpens = choice.openFrom | date %}
+    {% if dateCourseOpens > dateToday or choice.full == true %}
+      {% set canSubmit = false %}
+      {% set provider = providers[choice.providerCode] %}
+      {% set course = provider.courses[choice.courseCode] %}
+
+      {% if dateCourseOpens > dateToday %}
+        {% set cannotSubmitReasons = cannotSubmitReasons | push("you can only apply for " + course.name_and_code + " from " + (choice.openFrom | date("DDD"))) %}
+      {% else %}
+        {% set cannotSubmitReasons = cannotSubmitReasons | push("there are no places left on the " + course.name_and_code + " course") %}
+      {% endif %}
+
+    {% endif %}
+  {% endfor %}
+
+  {% if canSubmit %}
+    {{ govukButton({
+      text: "Continue"
+    }) if not closed }}
+  {% else %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if cannotSubmitReasons.length == 1 %}
```

Comment:
Think this needs to use the Nunjucks filter.
```suggestion
      {% if cannotSubmitReasons | length == 1 %}
```